Add download_data_from_download-o-o.sh

lkocman opened the pull request:
* Extract translatable files/metadata from download-o-o This is especially useful for layered projects (e.g. Leap >= 15.3), where buildresults of one or more layer do not contain extracted desktopfiles etc (needed by download_data.sh).
* Actual data is submitted in #14
* Original discussion about issues with newer Leap releases in https://progress.opensuse.org/issues/100023

Vogtinator commented on 51-xml/download_data_from_download-o-o.sh:
```diff
@@ -0,0 +1,117 @@
+            bzip2 $tgdir/$pkg-desktopfiles.tar
+        fi
+    fi
+    if [ ! -f "$tgdir/$pkg-appstream.tar.bz2" ]; then
```
AFAIK only the desktop file translations are actually used, the others were implemented for some possible future expansion which never happened. You might be able to save quite a bit of time.

lkocman committed:
Add download_data_from_download-o-o.sh

* Extract translatable files/metadata from download-o-o
  This is especially useful for layered projects (e.g. Leap >= 15.3),
  where buildresults of one or more layer do not contain extracted
  desktopfiles etc (needed by download_data.sh).

* Actual data is submitted in Leap 15.6 desktop-file-translations #14
  Original discussion about issues with newer Leap releases in
  https://progress.opensuse.org/issues/100023

lkocman committed:
Disable appstream and polkit extraction

lkocman commented:
Oky "rebased" and commented out fitlers for extraction of polkit and appstream files, that will cut some time. I'll keep the part for handling it, just in case we'd ever need it in the future.
